v3/controllerworks/city42/city42.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Controller Works city42",
+  "vendorId": "0x4357",
+  "productId": "0x0005",
+  "matrix": {"rows": 4, "cols": 12},
+  "menus": [ "qmk_rgb_matrix" ],
```

Comment:
Looks like in QMK not all effect are enabled. Please align QMK with VIA or VIA with QMK source

Reply:
I have added all of the animations to QMK in this PR (not yet approved):
[Add all RGB Matrix Animations to city42](https://github.com/qmk/qmk_firmware/pull/21992)

Reply:
The PR above with all of the RGB matrix animations was merged.